code/game/gamemodes/changeling/powers/lesserform.dm

```diff
@@ -21,6 +21,8 @@
 		to_chat(H, "<span class='warning'>We cannot perform this ability in this form!</span>")
 		return
 
+	var/datum/dna/DNA = changeling.GetDNA(user.dna.real_name)
+	user.dna = DNA ? DNA.Clone() : user.dna
```

Comment:
For consistency with the other changes I made and to bring this in-line with how cloning handles giving people their identities and such